thorlcr/thorutil/thmem.cpp

```diff
@@ -392,7 +411,11 @@ class CSpillableStream : public CSpillableStreamBase, implements IRowStream
         ++pos;
         return row;
     }
-    virtual void stop() { }
+    virtual void stop()
+    {
+        CThorArrayLockBlock block(rows);
```

Comment:
Should we check spillFile here, before acquiring the lock and calling clearSpillingCallback() ?  Not sure, just asking.